### ISSUE TITLE
hi. does this repo implements tinyVITS?

### ISSUE THREAD

**Issue by lucasjinreal** [rishikksh20/iSTFTNet-pytorch#11]
https://arxiv.org/abs/2206.00208

**Comment by rishikksh20** [comment]
Nope, tinyVITS also use similar architecture for Waveform decoder but this repo solely code iSTFTNet from this paper: https://arxiv.org/pdf/2203.02395.pdf
